intg-androidtv/apps.py

```diff
@@ -22,7 +22,16 @@
     "ARD Mediathek": {"url": "https://www.ardmediathek.de/"},
     "ZDF Mediathek": {"url": "https://www.zdf.de/filme"},
     "Kodi": {"url": "market://launch?id=org.xbmc.kodi"},
-}
+    "1und1": {"url": "1und1tv://1und1.tv"},
+    "Arte": {"url": "arte://display"},
+    "Google Play Store": {"url": "https://play.google.com/store/"},
+    "DVB-C/T/S": {"url": "market://launch?id=org.droidtv.playtv"},
+    "ATV Inputs": {"url": "market://launch?id=org.droidtv.channels"},
+    "ATV PlayFI": {"url": "market://launch?id=com.phorus.playfi.tv"},
+    "ATV Now on TV": {"url": "market://launch?id=org.droidtv.nettvrecommender"},
+    "ATV Media": {"url": "market://launch?id=org.droidtv.contentexplorer"},
+    "ATV Browser": {"url": "market://launch?id=com.vewd.core.browserui"},
+ }
```

Comment:
Linting issue https://github.com/unfoldedcircle/integration-androidtv/actions/runs/8891314506/job/24413032213?pr=43

You can check & format the code locally with Black: https://github.com/unfoldedcircle/integration-androidtv/blob/main/docs/code_guidelines.md#format-code

Most IDEs allow to configure "save-actions" to always format the code when it's saved.